.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,43 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: "[feature request]"
+labels: enhancement
+assignees: ''
+
+---
+
+------------------------------- delete below -------------------------------
+
+Thank you for raising a feature request! To get started, here's a few things to consider:
+
+# 1) Check existing requests:
```

Comment:
minor: use `#1` or `1)`

Don't mix the styles. :)

Reply:
Tried removing '#' but then figured out that it is a style marker to set the contents of that line to bold headline. Therefore, added it back.

Reply:
Ah, I see. It is markdown.